core/templates/dev/head/services/PlaythroughService.js

```diff
@@ -155,20 +154,14 @@ oppia.factory('PlaythroughService', [
             value: multipleIncorrectStateName.num_times_incorrect
           }
         };
-        return;
-      }
-
-      if (isCyclicStateTransitionsIssue()) {
+      } else if (isCyclicStateTransitionsIssue()) {
```

Comment:
I think this looks much better. Thanks for cleaning this up!

Reply:
No problem :)